src/viam/utils.py

```diff
@@ -351,3 +265,8 @@ def from_dm_from_extra(extra: Optional[Dict[str, Any]]) -> bool:
         return False
 
     return bool(extra["fromDataManagement"])
+
+
+_numpy_import_error_string = """MLModel support in Viam Python SDK requires the installation of optional
+dependencies: specifically, numpy.  Update your package using the extra [mlmodel]
+e.g. `pip install viam-sdk[mlmodel]` or the equivalent update in your dependency manager"""
```

Comment:
If we require `numpy` for all MLModel usage in our Py SDK, then we must use `required` instead of `optional`. What do you think of:

```suggestion
_numpy_import_error_string = """MLModel support in the Viam Python SDK requires the installation of an
additional dependency: numpy.  Update your package using the extra [mlmodel]
e.g. `pip install viam-sdk[mlmodel]` or the equivalent update in your dependency manager."""
```

Reply:
We refactor the string elsewhere but I replaced with your suggestions there